beta/src/pages/_app.tsx

```diff
@@ -30,20 +30,24 @@ export default function MyApp({Component, pageProps}: AppProps) {
   return (
     <AppShell>
       <Component {...pageProps} />
-      <Script
-        strategy="lazyOnload"
-        src={`https://www.googletagmanager.com/gtag/js?id=${process.env.NEXT_PUBLIC_GA_TRACKING_ID}`}
-      />
-      <Script id="google-analytics" strategy="lazyOnload">
-        {`
+      {process.env.NODE_ENV === 'production' && (
+        <>
```

Comment:
no need for a fragment here

Reply:
There are two script tags? That is why I added fragment 🤔.

Reply:
ahh I missed it. Ok then!

Reply:
:)